frontend/server/src/Controllers/Group.php

```diff
@@ -569,4 +569,74 @@ public static function getGroupScoreboardEditForTypeScript(
             'entrypoint' => 'group_scoreboard_contests',
         ];
     }
+
+    /**
+     * Utility function to create a new team group.
+     */
+    public static function createTeamGroup(
+        string $alias,
+        string $name,
+        string $description,
+        int $ownerId
```

Comment:
```suggestion
        int $ownerUserId
```

aquí estaría bueno clarificar qué tipo de ID es.

Reply:
Listo